packages/core/components/TutorialTooltip/TutorialTooltip.module.css

```diff
@@ -5,46 +5,65 @@
 
 .button-footer {
     display: flex;
+    justify-content: space-between;
+}
+
+.clear-button {
+    margin-left: auto;
+}
+
+.clear-button i {
+    color: var(--secondary-text-color);
+}
+
+.clear-button:hover, .clear-button:hover i {
+    background-color: var(--highlight-background-color);
+    color: var(--highlight-text-color);
 }
 
 .header {
+    align-items: center;
     display: flex;
 }
 
-.header > h6 {
-    margin-left: auto;
+.header > h4 {
+    font-weight: 500;
+    margin: 0;
+    padding: 0
 }
 
-.tutorial-container {
+.tutorial-container > div {
     background-color: var(--primary-background-color);
+    border-radius: var(--small-border-radius);
+    color: var(--primary-text-color);
```

Comment:
I think the tutorials are occasionally getting cut off a little on the far edges 
<img width="382" alt="image" src="https://github.com/AllenInstitute/aics-fms-file-explorer-app/assets/24441706/9f41030d-3f46-451f-b624-ad1f63c41c8d">

Reply:
Fixed _I think_

Reply:
I'll note this for the issue targeted for adding responsive design 👍  good catch